newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -109,78 +110,96 @@ export default function AnswerPage() {
     return null;
   }
 
+  const {final_dt: finalDate} = newdle;
+
   return (
     <div>
       <Grid container>
-        {error && (
+        {finalDate ? (
           <Grid.Row centered>
-            <Message error>
-              <p>Something went wrong while sending your answer:</p>
-              <code>{error}</code>
-            </Message>
-          </Grid.Row>
-        )}
-        {saved && (
-          <Grid.Row centered>
-            <Message success>
-              <p>Your answer has been saved!</p>
-            </Message>
+            <Grid.Column width={12}>
+              <Message
+                info
+                icon="info circle"
+                header="This newdle has already finished"
+                content="It is not possible to answer this newdle anymore. The author has already chosen a final date."
```

Comment:
Isn't "The author has already chosen a final date." clear from what we show below?